pimd/pim_igmp.c

```diff
@@ -479,17 +518,8 @@ int pim_igmp_packet(struct igmp_sock *igmp, char *buf, size_t len)
 	char from_str[INET_ADDRSTRLEN];
 	char to_str[INET_ADDRSTRLEN];
 
-	if (len < sizeof(*ip_hdr)) {
```

Comment:
It's not correct to move this check to `pim_igmp_verify_header`.
You're using header fields before the call to this function.
Please, move it back.

Reply:
@mobash-rasool please fix this and the next comment.

Reply:
Okay sure, will check and raise a new PR.